src/model/chart-model.ts

```diff
@@ -175,6 +175,39 @@ export type VisiblePriceScaleOptions = PriceScaleOptions;
  */
 export type OverlayPriceScaleOptions = Omit<PriceScaleOptions, 'visible' | 'autoScale'>;
 
+/**
+ * Determine how to exit the tracking mode.
+ *
+ * By default, mobile users will long press to deactivate the scroll and have the ability to check values and dates.
+ * Another press is required to activate the scroll, be able to move left/right, zoom, etc.
+ */
+export const enum TrackingExitMode {
+	/**
+	 * Tracking Mode will be deactivated on touch end event.
+	 */
+    OnTouchEnd,
+	/**
+	 * Tracking Mode will be deactivated on the next tap event.
+	 */
+    OnNextTap,
```

Comment:
```suggestion
	OnTouchEnd,
	/**
	 * Tracking Mode will be deactivated on the next tap event.
	 */
	OnNextTap,
```

looks like tabs vs spaces issue